### ISSUE TITLE
[BUG] Get-PnPSiteTemplate cannot export ClientSidePages

### ISSUE THREAD

**Issue by HeKr09** [pnp/powershell#2689]
### Reporting an Issue or Missing Feature
During executing Get-PnPSiteTemplate I receive an error.
After installing the older version 1.11.0 I did **not** receive the error.
Unfortunately after running the same script again with 1.12.0 (without uninstalling 1.11.0) the error vanished.
Can anybody help on this miracle?

### Expected behavior 
I expected that the cmdlet exports a xml.

### Actual behavior 
Currently the error is not present because of the above _solution_.

### Steps to reproduce behavior
`Get-PnPSiteTemplate -Out $fullPath -Schema LATEST -Encoding ([System.Text.Encoding]::UTF8) -Handlers Lists, RegionalSettings, SiteSettings, WebSettings, Fields, Files, Navigation, ContentTypes, PropertyBagEntries -Force -ListsToExtract $ListsToExtract`

### What is the version of the Cmdlet module you are running?
1.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by jenunn** [comment]
@HeKr09 have you tried adding the handlers "Pages" and "PageContents"?

I know that if you do not add those handlers, it's previously (and still?) jsut exporting the home.aspx page

**Comment by HeKr09** [comment]
@jenunn Yeah. Actually the complete code is as follows. I only send you the first line - my mistake :( I actually ran the second line.
```
if ($SkipAllClientSidePages) {
    Get-PnPSiteTemplate -Out $fullPath -Schema LATEST -Encoding ([System.Text.Encoding]::UTF8) -Handlers Lists, RegionalSettings, SiteSettings, WebSettings, Fields, Files, Navigation, ContentTypes, PropertyBagEntries -Force -ListsToExtract $ListsToExtract
}
else {
    Get-PnPSiteTemplate -Out $fullPath -Schema LATEST -Encoding ([System.Text.Encoding]::UTF8) -IncludeAllPages -Handlers Lists, Pages, PageContents, RegionalSettings, SiteSettings, WebSettings, Fields, Files, Navigation, ContentTypes, PropertyBagEntries -Force -ListsToExtract $ListsToExtract
}
```

**Comment by gautamdsheth** [comment]
Can you please try in another site collection or if possible in another tenant ? Maybe you are trying this in a classic site collection.

Looks like some issue specific to your machine, as am not able to reproduce this. 
Am moving this to a discussion forum.
